src/CloudEventFunctionWrapper.php

```diff
@@ -42,6 +46,12 @@ class CloudEventFunctionWrapper extends FunctionWrapper
         'time'
     ];
 
+    public function __construct(callable $function, bool $marshalToCloudEventInterface = false)
+    {
+        $this->marshalToCloudEventInterface = $marshalToCloudEventInterface;
+        parent::__construct($function);
```

Comment:
This is not optimal because you're changing the constructor of `FunctionsWrapper`, which has a `$signature` parameter that is now unavailable.

This is breaking backwards compatibility and also making that parameter unavailable to users.

Reply:
I can add back the `$signature` optional parameter.